README.md

```diff
@@ -163,6 +163,74 @@ python buildbot/compile.py -o build
 > **_Note:_**
   The instructions differ for the host and riscv. The instructions mentioned above are for `riscv` target.
 
+### Compiling official Intel oneAPI Base Toolkit
+The official Intel OneAPI Base Toolkit can be obtained by visiting the following link: [Intel OneAPI Base Toolkit Download](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html). On this page, specify the operating system, the type of installer needed, and the desired version to access the download options. The initial download is for the installer application files only. The installer will acquire all the tools during the installation process. From the console, locate the downloaded install file.
+
+```sh
+# To launch the GUI installer as the root
+sudo sh ./<installer>.sh
+```
+Or
+```sh
+# To launch the GUI installer as the current user.
+sh ./<installer>.sh
+```
+
+Follow the instructions in the installer. And explore the Get Started Guide to get more information.
+
+For example, for Linux, online installer and version 2023.2.0, follow the instructions below:
+
+```sh
+wget https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397.sh
+
+sudo sh ./l_BaseKit_p_2023.2.0.49397.sh
+```
+
+Now to compile the vector add from oneAPI samples mentioned in previous section, using the official Intel base toolkit, set the environment variables and follow the stepsgiven below:
+
+```sh
+export OCL_ICD_FILENAMES=$ONEAPI_CON_KIT_INSTALL_DIR/lib/libCL.so
+export LD_LIBRARY_PATH=/path/to/intel/oneapi/compiler/2023.2.0/linux/lib:/path/to/intel/oneapi/compiler/2023.2.0/linux/compiler/lib/intel64_lin:/path/to/intel/oneapi/compiler/2023.2.0/linux/compiler/lib/:$LD_LIBRARY_PATH
+export ONEAPI_DEVICE_SELECTOR="*:acc"
+
+/path/to/intel/oneapi/compiler/2023.2.0/linux/bin-llvm/clang++ -fsycl /path/to/oneAPI-samples/DirectProgramming/C++SYCL/DenseLinearAlgebra/vector-add/src/vector-add-buffers.cpp -o vector-add-buffers
+CA_HAL_DEBUG=1 SYCL_CONFIG_FILE_NAME=
```

Comment:
the does not need null.cfg then?

Reply:
No. I'll update the comment, if mentioned elsewhere. I think someone mentioned that if we add null.cfg here that would mean we need to have a file named null.cgf. Just leaving it empty works same way.